docs/languages/csharp/modern-events.md

```diff
@@ -1,12 +1,126 @@
 # The Updated .NET Core Event Pattern
 
+By [Bill Wagner](https://github.com/BillWagner)
+
 [Previous](event-pattern.md)
 
-This article will cover the updates to the
-.NET event pattern that enables new and different interfaces
+The previous article discussed the most common event patterns. .NET
+Core has a more relaxed pattern. In this version, the 
+`EventHandler<TEventArgs>` definition no longer has the constraint that
+`TEventArgs` must be a class derived from System.EventArgs.
```

Comment:
Mind using the backtick formatting here?  System.EventArgs ---> `System.EventArgs`